### ISSUE TITLE
Stuck on Checking for Tor Status

### ISSUE THREAD

**Issue by McFarTech** [bissisoft/torall#20]
Hello. I have installed the TorAll on the latest version of Kali. When i start it i am getting stuck on the Checking for Tor Status. 

I am unable to get out to the internet at all unless i reboot. Tor Browser works fine, internet works fine but TorAll not so good. Any suggestions on what might be wrong?

**Comment by McFarTech** [comment]
_______________          ____  ___  ___
        /____    _____//         /    \ \  \ \  \
            /   //___V/   __ __ /  /\  \ \  \ \  \
           /   //  _  \. /  ,._|  //_\  \ \  \ \  \
         _/   ((  (O)  ))  // /  ______  \ \  \_\  \_
        /_____//\_____//\_//  \_//     \_// \__//\__//
        v2.0 - github.com/bissisoft/torall
    

        STARTING TorAll...
        Disabling IPv6...
        Switching nameservers...
        Spoofing mac addresses...
        Starting new tor daemon...
        Backing up iptables rules...
        Setting up iptables rules...
        Checking tor network status...

**Comment by S0h1n1** [comment]
@McFarTech  Thank you for feedback. I just did a fresh Kali install and could not reproduce this issue. I did come up to another problem actually a dependency issue of iptables (because i did a minimal Kali install that was missing iptables -- i will report it as [another new issue](https://github.com/bissisoft/torall/issues/21)). After i installed the missing iptables (sudo apt install iptables) everything worked just fine. Is there more info that you can provide? Are you running Kali in a virtual machine or on 'metal'? Have you tried to run torall again? From the looks of it, the software was waiting on a reply from the tor network and it took a long time for that status reply... maybe the tor network was very busy/slow or the tor status service was down at that very moment.

**Comment by S0h1n1** [comment]
![Screenshot_2023-07-10_10-15-29](https://github.com/bissisoft/torall/assets/99693942/3e7bff5d-a970-4e18-9386-c4f422a8446b)

**Comment by McFarTech** [comment]
Thanks mate. I did a fresh install of latest Kali with all packages. Same issues.

TorGhost does work though. I would prefer to use TorAll 

I am running Kali under VMware ESXi

**Comment by bissito** [comment]
@McFarTech @S0h1n1 Thank you guys. We will investigate this issue! By the way, it is time for a new and better version coming out soon anyway :wink:
